Automattic/wp-calypso
# WordAds: Point Jetpack and Atomic users to Jetpack settings rather than Marketing.

#### Changes proposed in this Pull Request

* Remove JetpackAds settings under Marketing -> Traffic; in my searches this component was only used once and only on this page, so I've removed the component as well.
* Redirect Jetpack and Atomic users to the Jetpack Ads settings when viewing the Ads Settings dashboard.

#### Testing instructions

Switch to this PR and test this flow on the following types of sites:

* WP.com Premium Simple
* WP.com Business Non-atomic
* WP.com Business Atomic
* Jetpack Security Daily

You're looking for changes to how we navigate to WordAds settings:

* Starting at `/earn`, set up a WordAds account for your site.
* If on a WP.com Premium Simple or WP.com Business Non-Atomic site, ads settings will remain under `/earn/ads-settings`

<img width="1146" alt="Screen Shot 2021-04-05 at 12 16 39 PM" src="https://user-images.githubusercontent.com/2124984/113596768-c9887480-9608-11eb-869d-6a068f2d2a60.png">

* If on a WP.com Business Atomic or Jetpack Security Daily site, you'll see a "Manage ad placements" link under Settings:

<img width="1112" alt="Screen Shot 2021-04-05 at 12 17 23 PM" src="https://user-images.githubusercontent.com/2124984/113596850-e329bc00-9608-11eb-81fc-bcb5803dac2c.png">

* If you click on "Manage ad placements", you should be directed to the Jetpack Traffic Settings under `{site}/wp-admin` for WP.com Business Atomic and Jetpack Security Daily sites:

<img width="1114" alt="Screen Shot 2021-04-05 at 12 15 23 PM" src="https://user-images.githubusercontent.com/2124984/113597639-fee19200-9609-11eb-9ad8-318b1d4ae8b5.png">

Fixes #49144
